README.md

```diff
@@ -5,7 +5,7 @@ This gem provides an implementation of the [time aware polyline](https://medium.
 ## Encoder
 ​
 ```ruby
-require 'time-aware-polyline'
```

Comment:
Weird that this got changes to underscores and then back to dashes. 🤷

Reply:
I think this might have just gotten missed in the original change but im not sure.